### ISSUE TITLE
UI Help Tips

### ISSUE THREAD

**Issue by daniel-cherry** [cloud-elements/devportal#398]
When using the skeletor console, I should see help tips throughout the various UIs

**Comment by daniel-cherry** [comment]
We need specifics on where help tips are needed, what the help tip should say, etc...

**Comment by davewilks** [comment]
I agree and this could be ongoing so an estimate might not be useful.
